stacks/container-apps/cms-alarms.yml

```diff
@@ -0,0 +1,115 @@
+    Type: String
+  EnvironmentTypeAbbreviation:
+    Type: String
+  AudioSqsCallbackName:
+    Type: AWS::SSM::Parameter::Value<String>
+  ImageSqsCallbackName:
+    Type: AWS::SSM::Parameter::Value<String>
+Resources:
+  # callback queues are being slow
+  CmsAudioCallbackSlowAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: !Sub "[CMS][Worker][SQS] ${EnvironmentType} Audio Callbacks Slow"
+      AlarmActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      InsufficientDataActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      OKActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      AlarmDescription: Audio callbacks are behind
+      ComparisonOperator: GreaterThanThreshold
+      EvaluationPeriods: 3
+      MetricName: ApproximateAgeOfOldestMessage
+      Namespace: AWS/SQS
+      Period: 60
+      Statistic: Average
+      Threshold: 60
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: QueueName
+          Value: !Ref AudioSqsCallbackName
+  CmsImageCallbackSlowAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: !Sub "[CMS][Worker][SQS] ${EnvironmentType} Image Callbacks Slow"
+      AlarmActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      InsufficientDataActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      OKActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      AlarmDescription: Image callbacks are behind
+      ComparisonOperator: GreaterThanThreshold
+      EvaluationPeriods: 3
+      MetricName: ApproximateAgeOfOldestMessage
+      Namespace: AWS/SQS
+      Period: 60
+      Statistic: Average
+      Threshold: 60
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: QueueName
+          Value: !Ref ImageSqsCallbackName
+  # callback queues likely aren't processing
+  CmsAudioCallbackFatalAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: !Sub "[CMS][Worker][SQS] ${EnvironmentType} Audio Callbacks Not Processing"
+      AlarmActions:
+        - !Ref OpsFatalMessagesSnsTopicArn
+      InsufficientDataActions:
+        - !Ref OpsFatalMessagesSnsTopicArn
+      OKActions:
+        - !Ref OpsFatalMessagesSnsTopicArn
+      AlarmDescription: Audio callbacks are maybe not processing
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      EvaluationPeriods: 1
+      MetricName: ApproximateAgeOfOldestMessage
+      Namespace: AWS/SQS
+      Period: 60
+      Statistic: Average
+      Threshold: 300
```

Comment:
And if _anything_ in the queue is older than 5 minutes ... fatal!

(Looking through the past month of data, we get a few minutes with datapoints slightly above 60, but nothing anywhere near 300).